packages/dashboard/e2e/tests/trendCursors/trendCursors.spec.ts

```diff
@@ -185,8 +185,18 @@ test('can drag a trend cursor', async ({ page }) => {
   expect(initialTrendCursorDateString).not.toEqual(
     updatedTrendCursorDateString
   );
-  const initialDate = new Date(initialTrendCursorDateString);
-  const updatedDate = new Date(updatedTrendCursorDateString);
+  const initialDate = new Date(
+    initialTrendCursorDateString
+      .split('\n')
+      .join('T')
+      .replace(/(\d+)\/(\d+)\/(\d+)/, '$3-$2-$1')
```

Comment:
does the date-fns library not support this?

Reply:
supports. updated.